manifest/content.xml

```diff
@@ -73,4 +73,27 @@
     <button />
   </form>
 
+  <!--
+    Tabs
+      Required childr elements: options, option, tab-content, tab
+      Each option element requires a single content:text child element
+      The number of tab elements should be the same of the number of option element
+  -->
+  <tabs>
+    <options>
+      <option><content:text i18n-id="{{uuid}}">Option A</content:text></option>
```

Comment:
can we simplify this XML by integrating the options directly into the tab node itself. something like:

```
<tabs>
  <tab>
    <label>
      <text i18n-id="{{uuid}}">Option A</text>
    </label>
    <image />
    <text />
  </tab>
</tabs>